Update MESSAGE_CONSTRAINTS to non-negative integer values

Jh123x opened the pull request:
- [x] Update MESSAGE_CONSTRAINTS to non-negative integer values
- [x] Update UG as well

Jh123x committed:
Update MESSAGE_CONSTRAINTS to non-negative integer vakues

Update MESSAGE_CONSTRAINTS to non-negative integer vakues

Jh123x committed:
Update UG to Match Macronutrient class

aidoxe-123 reviewed (changes requested):
The index should be positive instead of non-negative, isn't it?

Jh123x committed:
Add Message to inform user about values of Macronutrients

Add Message:

Values of `PROTEIN`, `FATS` and `CARBS` must be a non-negative integer.<br>

Revert Erroneous changes made to Index

Jh123x commented:
> The index should be positive instead of non-negative, isn't it?

Nice catch. 
Idk what I was reading x.x

aidoxe-123 reviewed (approved):
LGTM
